### DISCUSSION TITLE
3D panel creation locally

### DISCUSSION THREAD

**Discussion by pppaw** [foxglove/community#818] [General]
Hello,
I want to create 3D panel locally. Can anyone tells how to create same 3D foxglove panel locally. Which files need to download  and which packages need to install from the foxglove github ?
 I am very new to Foxglove studio.

**Comment by jtbandes** [comment]
The 3D panel is not designed for use outside of the foxglove/studio repository. It should be technically possible; since the code is open source you may adapt it as needed, but we do not provide explicit support for this kind of development.
